### DISCUSSION TITLE
2024/11/07/%E7%BA%BF%E6%80%A7%E4%BB%A3%E6%95%B04-5%E7%AB%A0/

### DISCUSSION THREAD

**Discussion by giscus** [ShwStone/shwblog#79] [Announcements]
# 2024/11/07/%E7%BA%BF%E6%80%A7%E4%BB%A3%E6%95%B04-5%E7%AB%A0/

https://m.shwstone.cn/2024/11/07/%E7%BA%BF%E6%80%A7%E4%BB%A3%E6%95%B04-5%E7%AB%A0/

**Comment by ChordGo** [comment]
很好奇这些公式是纯手打的吗，还是有什么高效输入LaTeX的工具？

**Comment by ShwStone** [comment]
事实上手打是最高效的

可以使用一些snippets辅助输入
